package.json

```diff
@@ -34,12 +34,12 @@
     "@aws-sdk/client-s3": "^3.731.1",
     "@fingerprintjs/fingerprintjs-pro-server-api": "^6.1.0",
     "@nestjs/cli": "^10.4.9",
-    "@nestjs/common": "^10.4.15",
+    "@nestjs/common": "^11.0.3",
     "@nestjs/config": "^4.0.0",
-    "@nestjs/core": "^10.4.15",
-    "@nestjs/platform-express": "^10.4.15",
+    "@nestjs/core": "^11.0.3",
+    "@nestjs/platform-express": "^11.0.3",
     "@nestjs/schedule": "^4.1.2",
-    "@nestjs/serve-static": "^4.0.2",
+    "@nestjs/serve-static": "^5.0.1",
```

Comment:
We need to update this here as the package otherwise prevents startup. Internally, it has an [unexpected `*` path](https://github.com/nestjs/serve-static/blob/4.0.2/lib/serve-static.constants.ts#L4), which goes against the [new path-to-regexp requirements](https://github.com/pillarjs/path-to-regexp?tab=readme-ov-file#unexpected--or-). This stems from [Express v5](https://docs.nestjs.com/migration-guide#express-v5).